src/controllers/commentsController.js

```diff
@@ -10,71 +10,75 @@ import notificationsController from './notification';
 const { CommentNotification } = notificationsController;
 const { Comments, travelRequests } = Models;
 const {
-  Op
+    Op
 } = Sequelize;
 class commentsController {
-  async addComment(req, res) {
-    try {
-      const {
-        query: {
-          requestId
-        },
-        body: {
-          comment
-        },
-        user: {
-          id, email, preferedLang
-        }
-      } = req;
-      const found = await travelRequests.findOne({
-        where: {
-          id: requestId,
-          [Op.or]: [
-            { userID: id },
-            { manager: email },
-          ]
-        }
-      });
-      if (!found) {
-        return errorResponse(res, 403, setLanguage(preferedLang).__('commentSorry'));
-      }
-      const newComment = await Comments.create({
-        id: uuid(),
-        userID: id,
-        requestId,
-        comment
-      });
```

Comment:
@RusimbiP for example you copied this block of code, deleted it and pasted it below but there is no single change both blocks are identical, so I was asking why you did that.

Reply:
Ooooh, It is prettier

Reply:
The vscode extension